src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,3 @@
-export const PostList = () => (
-  <>Put the list here</>
-);
+
+/* export const PostList = () => (
```

Comment:
No implementation for the PostList component. According to the task, it should render a static list of posts. Each post should have a user and comments associated with it. Split the app into components based on CSS blocks and style them accordingly. The List components should render one Info component per object in the array. The Info component should render the data for each object.